### PR TITLE
feat: support indent level as option

### LINKED ISSUE #240: Feature Proposal - Disable Indent
I'd like the option to disable the indent.

```
  Some descriptive text
  Usage:
    $ application
```

Is the default, and I'd like to make it indented to 0.

Proposal:
Add `indent` as an option with a default value of `2`.

I have submitted a PR for this.

### PR DESCRIPTION
**Pull request by dylanarmstrong**
Fixes #240

### CODE REVIEW

**Comment by dylanarmstrong** [comment]
> The `index.test-d.ts` file needs to be updated.

Thanks, but per the attached issue, I’m unsure the project is going to actually use this PR. If I get approval there that this is wanted from maintainers, I’ll update this PR.

**Comment by dylanarmstrong** [comment]
Hi, is there any update on this?

File: readme.md [hunk 1]
```diff
@@ -299,6 +299,13 @@ Default: `true`
 
 Whether to allow unknown flags or not.
 
+##### helpIndent
+
+Type `number`\
+Default: `2`
+
+Indentation to use on helpText.
```
Comment (sindresorhus) [inline readme.md:307]: Needs a better description.

File: source/index.d.ts [hunk 2]
```diff
@@ -307,6 +307,13 @@ export type Options<Flags extends AnyFlags> = {
 	@default true
 	*/
 	readonly allowUnknownFlags?: boolean;
+
+	/**
+	Indentation to use on helpText.
+
+	@default true
```
Comment (sindresorhus) [inline source/index.d.ts:314]: Incorrect default.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: source/index.d.ts (hunk 2 above)
Comment (dylanarmstrong) [reply source/index.d.ts]: Updated

File: readme.md (hunk 1 above)
Comment (dylanarmstrong) [reply readme.md]: Updated

**Comment by dylanarmstrong** [comment]
Thanks for the review @sindresorhus, I've updated the PR.

**Pull request merged**